### PR TITLE
media-sound/wav2json: new ebuild

### PR DESCRIPTION
**Pull request by akreal**

### CODE REVIEW

File: media-sound/wav2json/wav2json-0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+S=${WORKDIR}/${PN}-${COMMIT_ID}/build
+
+LICENSE="GPL-1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-libs/boost
```
Comment (SoapGentoo) [inline media-sound/wav2json/wav2json-0.4.ebuild]: `:=` sub-slot operator missing, always required for boost!

File: media-sound/wav2json/wav2json-0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	media-libs/libsndfile
+"
+RDEPEND="${DEPEND}"
+
+src_install() {
+	dobin ../bin/Linux/wav2json
```
Comment (SoapGentoo) [inline media-sound/wav2json/wav2json-0.4.ebuild]: are you sure it doesn't install any other stuff with `make install` or something? Just asking, haven't checked the BUILD_DIR myself

**Comment by monsieurp** [comment]
Thanks for the PR. Could you give a little bit more background as to what this package is meant to be solving? Clearly, "new ebuild" is obvious and not sufficient to describe what the package is really about.

File: media-sound/wav2json/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sound@gentoo.org</email>
```
Comment (mgorny) [inline media-sound/wav2json/metadata.xml]: Please do not add Gentoo projects or developers as maintainers of your packages if you haven't talked to them first. Instead, add yourself and the proxy-maint project.

File: media-sound/wav2json/wav2json-0.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+EAPI=6
+
+COMMIT_ID="e412923f1f792408e8ae1096ca40fb5307ddfc74"
+DESCRIPTION="Generate waveformjs.org compatible json data out of wav files"
+HOMEPAGE="https://github.com/beschulz/wav2json"
+SRC_URI="https://github.com/beschulz/wav2json/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}-${COMMIT_ID}/build
+
+LICENSE="GPL-1"
```
Comment (mgorny) [inline media-sound/wav2json/wav2json-0.4.ebuild]: I really doubt the author means GPL-1. Could you confirm that with him?

File: media-sound/wav2json/wav2json-0.4.ebuild (hunk 1 above)
Comment (akreal) [inline media-sound/wav2json/wav2json-0.4.ebuild]: Fixed.

File: media-sound/wav2json/wav2json-0.4.ebuild (hunk 2 above)
Comment (akreal) [inline media-sound/wav2json/wav2json-0.4.ebuild]: Yes, this binary is the only item it brings (and there is no `make install`).

File: media-sound/wav2json/metadata.xml (hunk 3 above)
Comment (akreal) [inline media-sound/wav2json/metadata.xml]: Done.

File: media-sound/wav2json/wav2json-0.4.ebuild (hunk 4 above)
Comment (akreal) [inline media-sound/wav2json/wav2json-0.4.ebuild]: You are right, author meant GPL-2. Fixed.

**Comment by akreal** [comment]
@monsieurp this package installs `wav2json` utility, which takes audio in WAV format and outputs waveform as floating points arrays in JSON. Primary use for such JSON are Web apps (e.g. using waveform.js or wavesurfer.js libraries) or other cases when somebody wants to visualize sound without having to process actual audio data.

File: media-sound/wav2json/wav2json-0.4.ebuild (hunk 2 above)
Comment (SoapZA) [inline media-sound/wav2json/wav2json-0.4.ebuild]: perfect, LGTM 😄

**Comment by akreal** [comment]
Thank you!

**Pull request closed without merging**